tests/pipelines/animatediff/test_animatediff.py

```diff
@@ -279,7 +283,7 @@ def test_prompt_embeds(self):
 
         inputs = self.get_dummy_inputs(torch_device)
         inputs.pop("prompt")
-        inputs["prompt_embeds"] = torch.randn((1, 4, 32), device=torch_device)
+        inputs["prompt_embeds"] = torch.randn((1, 4, pipe.text_encoder.config.hidden_size), device=torch_device)
```

Comment:
I think this is the more correct thing to do. 32 was hardcoded because that was the cross_attention_dim/hidden_size earlier but ideally it should be read from the config for this test.